build.gradle

```diff
@@ -246,7 +246,8 @@ ext.externalDependency = [
   "reflections" : "org.reflections:reflections:0.9.10",
   "embeddedProcess": "de.flapdoodle.embed:de.flapdoodle.embed.process:1.50.2",
   "testMysqlServer": "com.wix:wix-embedded-mysql:1.1.0",
-  "flyway": "org.flywaydb:flyway-core:3.2.1"
+  "flyway": "org.flywaydb:flyway-core:3.2.1",
+  "oltu": "org.apache.oltu.oauth2:org.apache.oltu.oauth2.client:1.0.1"
```

Comment:
Checked Artifactory and it does not exist -- do we need dependencies in internal artifactory?
As per version, used 1.0.1 when it was the latest when I first started working. Will update to 1.0.2.